data/event-schemas/schema/m.call.sdp_stream_metadata_changed.yaml

```
@@ -0,0 +1,21 @@
+type: object
+x-addedInMatrixVersion: "1.10"
```

sadly this missed the boat

```suggestion
x-addedInMatrixVersion: "1.11"
```